src/utilities/valueFactory.js

```diff
@@ -20,25 +20,33 @@ const newValue = (
   component,
 })
 
-export const newLiteralValue = (property, literal, lang = defaultLanguageId) =>
+export const newLiteralValue = (property, literal, lang) =>
```

Comment:
Note: this method is always passed a language, so we do not need defaultLanguageId here